api/notes/notesRouter.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+
+const Notes = require('./notesModel.js');
+
+const router = express.Router();
+
+router.get('/notes', (req, res) => {
+  Notes.getNotes()
+    .then((notes) => res.status(200).json(notes))
+    .catch((error) => {
+      console.log('error in Note Router GET', error);
```

Comment:
Overall it looks really clean, I would just clean up some of the console logs that may be unnecessary on the main branch.

Reply:
Doing it now. Thanks for reminding me.